google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1283,6 +1295,15 @@ public String getLocation() {
     return location;
   }
 
+  /**
+   * Returns the bucket's locationType.
+   *
+   * @see <a href="https://cloud.google.com/storage/docs/bucket-locations">Bucket locationType</a>
```

Comment:
Please update this to:
`<a href="https://cloud.google.com/storage/docs/bucket-locations">Bucket LocationType</a>`

Reply:
done